variants/icarus/variant.h

```diff
@@ -0,0 +1,30 @@
+// Icarus has a 1.3 inch OLED Screen
+#define USCREEN_SSD106
```

Comment:
That looks like a typo. you sure?

Reply:
it was in the seeedstudio esp32s3 variant.h and im using a similar screen, the firmware works so no issues there haha

Reply:
That is not really authoritative ... :-) 

I looked it up, that's definitely a typo. Should be SCREEN_SSD1306 (or any other screen model, these are usually auto detected) and you define them only if you have problem with autodetect. May be the reason it works for you despite the typo.

Reply:
ah ok fixed :)